static/app/stores/groupStore.tsx

```diff
@@ -358,14 +358,20 @@ const storeConfig: Reflux.StoreDefinition & Internals & GroupStoreInterface = {
   },
 
   onDeleteSuccess(_changeId, itemIds, _response) {
+    const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).toString();
```

Comment:
do we want `toString` or do we want
```suggestion
    const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).join(', ');
```